packages/client/src/store/selectors/bookings/__tests__/bookings.test.ts

```diff
@@ -349,5 +353,108 @@ describe("Selectors ->", () => {
         },
       ]);
     });
+
+    test.only("should sort the slots (by date, and by time intraday)", () => {
```

Comment:
I guess this `only` was not supposed to end up in the final commit.

Reply:
Not at all...on it